### PR TITLE
Test / Update compute version to azure-mgmt-compute==4.6.0

### PR DESCRIPTION
**Pull request by adewaleo**
Test azure/mgmt/compute API Version 2019-03-01

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by adewaleo** [comment]
These tests failed:


test_vm_commands.VMCustomImageTest.test_vm_custom_image_name_conflict
test_vm_commands.VMDiskAttachDetachTest.test_vm_ultra_ssd_disk_update
test_vm_commands.VMDiskAttachDetachTest.test_vm_ultra_ssd_storage_sku
test_vm_commands.VMDiskAttachDetachTest.test_vmss_ultra_ssd_storage_sku
test_vm_commands.VMSSVMsScenarioTest.test_vmss_vms
test_vm_commands.VMManagedDiskScenarioTest.test_vm_managed_disk
test_vm_commands.MSIScenarioTest.test_vmss_explicit_msi
test_vm_commands.VMGalleryImage.test_gallery_e2e

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py [hunk 1]
```diff
@@ -392,6 +392,9 @@ def load_arguments(self, _):
             c.argument('parameters', nargs='+', help="space-separated parameters in the format of '[name=]value'")
             c.argument('scripts', nargs='+', help="script lines separated by whites spaces. Use @{file} to load from a file")
 
+        with self.argument_context('{} stop'.format(scope)) as c:
+            c.argument('skip_shutdown', action='store_true', help='Skip shutdown and power-off immediately.')
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: should have minimum api-version

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py [hunk 2]
```diff
@@ -587,12 +590,15 @@ def load_arguments(self, _):
         c.argument('exclude_from_latest', arg_type=get_three_state_flag(), help='The flag means that if it is set to true, people deploying VMs with version omitted will not use this version.')
         c.argument('version', help='image version')
         c.argument('end_of_life_date', help="the end of life date, e.g. '2020-12-31'")
+        c.argument('storage_account_type', help="The default storage account type to be used per region. To set regional storage account types, use --target-regions", arg_type=get_enum_type(["Standard_LRS", "Standard_ZRS"]))
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_params.py]: same here

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py [hunk 3]
```diff
@@ -1356,18 +1356,55 @@ def process_remove_identity_namespace(cmd, namespace):
...
+                try:
+                    replica_count = int(parts[1])   # raises ValueError if this is not a replica count, try other order.
+                    storage_account_type = parts[2]
+                    if storage_account_type not in storage_account_types_list:
+                        raise CLIError("usage error: {} is an invalid target region argument. The third part is "
+                                       "not a valid storage account type. Storage account types must be one of {}."
+                                       .format(t, storage_account_types_str))
+                except ValueError:
+                    try:
+                        replica_count = int(parts[2])  # raises ValueError if neither part is an integer replica count.
```
Comment (yugangw-msft) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py]: it is nice to support both, but let us comply to the help of `<region>[=<replica count>][=<storage account type>]`

**Pull request merged**